astropy/io/misc/hdf5.py

```diff
@@ -214,6 +218,45 @@ def _encode_mixins(tbl):
     return encode_tbl
 
 
+def convert_dtype(table, character_as_bytes, copy=False):
+    """
+    Converts bytestring columns (dtype.kind='S') to unicode (dtype.kind='U') and
+    unicode columns (dtype.kind='U') to bytestring (dtype.kind='S').
+
+    Parameters:
+    -----------
+    table: `~astropy.table.Table`
+         Data table in which changes are to be made.
+    character_as_bytes: boolean
+         True if columns are to be converted to bytestrings from unicode.
+         False if columns are to be converted to unicode from bytestrings.
```

Comment:
The indentation isn't quite right in this docstring, there should be four spaces before these lines, not five

Also, add single backticks around ``True`` and ``False``